helpers/address.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package helpers
+
+import (
+	"github.com/decred/dcrd/chaincfg/v2"
+	"github.com/decred/dcrd/dcrec"
+	"github.com/decred/dcrd/dcrutil/v2"
+	"github.com/decred/dcrd/hdkeychain/v2"
+)
+
+// DCRutilAddressFromExtendedKey returns a dcrutil Address object for extended
```

Comment:
This comment does little more than enumerate the function params and return type, it adds more noise than useful information. It could be useful to explain why this function exists, when it should be used, or mention some of the internal implementation details (`Hash160`, `Secp256k1`).

Also I think dcrutil should be capitalised as `DCRUtil`

Reply:
What do you think about the message now?

Reply:
Great :+1: